src/helpers.js

```diff
@@ -52,6 +56,62 @@ const inputCountriesToRuleCountries = inputCountries => {
     return ruleCountries
 }
 
+const getLevelOneCategory = cat =>
+    cat.split('-')[0]
+
+const getToFixedDecimal = num =>
+    parseFloat(num.toFixed(2))
+
+
+
+
+const getSuggestedPricingBoundsV1 = ({ minByCategory, countryTiersCoefficients, audienceInput }) => {
+    const { inputs } = audienceInput
+    const { location = {}, categories = {}, publishers = {}, advanced = {} } = inputs
+    const minCategoryCpm = Math.min(...Object.values(minByCategory))
+
+    const selectedCategoriesMinCpms = Object.values(Object.fromEntries(Object.entries(minByCategory).filter(([key, value]) => {
+
+        const inSelected = !categories.apply.includes('in') || !categories.in || !categories.in.length || categories.in.some(c => c === 'ALL' || (key === getLevelOneCategory(c)))
```

Comment:
split those to multiple lines?

Reply:
better no it will look like:
``` 
 const haNoInCategoriesInRules = !categories.apply.includes('in') || !categories.in || !categories.in.length
        const hasAppliedInCategories = categories.apply.includes('in') && categories.in && categories.in.length && categories.in.some(c => c === 'ALL' || (key === getLevelOneCategory(c)))
        const inSelected = hasAppliedInCategories || haNoInCategoriesInRules
        const notExcluded = !categories.apply.includes('nin') || !(categories.nin || []).some(c => key === getLevelOneCategory(c))

        return inSelected && notExcluded
```